sympy/physics/units/dimensions.py

```diff
@@ -579,6 +579,12 @@ def list_can_dims(self):
 
         List all canonical dimension names.
         """
+        SymPyDeprecationWarning(
+            deprecated_since_version="1.10",
+            issue=22953,
+            feature="list_can_dims",
+            useinstead="Will be removed",
```

Comment:
I think that the useinstead can be more helpful than this.

Reply:
Alright what do you suggest @oscarbenjamin ?

Reply:
True, i'm not sure what "Will be removed" conveys to the user under the label `useinstead`

Reply:
As it is my suggestion:

The whole thing seems to be some matrix-fiddling exercise and the alternative solution is most likely pretty long. The option is then to leave it empty, which is maybe worse?

(Third option: "The source.")

I think the main problem is that noone really has a good idea how this works. I really just went for the "DO NOT USE" and "useless" comments. So it seems like these functions were introduced in good spirit, but later on it turned out that one should do things in a *completely* different way and that these functions didn't make sense anymore. 

So while I agree it is a "bad" message, I am not sure anyone can come up with a better.

Reply:
Or, maybe @Upabjojr remembers the details why these methods are useless?

Reply:
All this was introduced in  #13287 and some of the methods were deprecated, others just had the "useless" comment added.